disks-and-volumes.md

```diff
@@ -258,6 +258,43 @@ spec:
         claimName: mypvc
 ```
 
+### Ephemeral Volume
+
+Ephemeral volume is a volume that expects a persistent volume as it's backing
```

Comment:
I took a stab a rewording this a bit. I just thought from the user's perspective that I would want to know a little bit more about where the ephemeral image lives (local storage) and what creates/destroys it (kubevirt during start/stop flow)

"An ephemeral volume is a local COW (copy on write) image that uses a PVC volume as a read-only backing store. With an ephemeral volume, the PVC backing store is never mutated. Instead all writes are stored on the ephemeral image which exists on local storage. KubeVirt dynamically generates the ephemeral images associated with a VM when the VM starts, and discards the ephemeral images when the VM stops."

Reply:
One nit I have with the reword is the direct reference to PVC as a backing volume. If we expand the feature to support more backing volume types, the paragraph would have to be changed for each such change.

Reply:
maybe replace 'PVC volume' with "network volume' then.  It doesn't matter much to me.

The big point here is to make sure that it's obvious to the user that the ephemeral disk is a kubevirt managed, dynamically generated disk that exists on local storage which acts as a COW disk for a read only backing volume.

Reply:
I would also not mind if we update the list of supported volume types and explicitly mention them here. It is what a user is interested in. Also adding a link to http://www.kubevirt.io/api-reference/master/definitions.html#_v1_ephemeralvolumesource could be considered.